examples/next-graphql/app/layout.tsx

```diff
@@ -2,9 +2,9 @@ import type { Metadata } from "next";
 import { Inter } from "next/font/google";
 import "./globals.css";
```

Comment:
Can we order it to have the CSS imports after JS stuff? 🙏

Reply:
fixed